build.sbt

```diff
@@ -102,6 +102,8 @@ scalacOptions in (Compile, console) ~= (_.filterNot(
   )
 ))
 
+scalafmtOnCompile := true
```

Comment:
why doing this? It's explicitly discouraged on scalafmt site for good reasons IMO

Reply:
I keep forgetting about formatting the code. I missed the warning in docs. Flag removed - I configured my IDE to do the job :)